<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchTab</class>
 <widget class="QWidget" name="SearchTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Sea&amp;rch: </string>
         </property>
         <property name="buddy">
          <cstring>searchLineEdit_3</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchLineEdit_3"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QComboBox" name="categoryComboBox_3">
         <item>
          <property name="text">
           <string>Title</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Abstract</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Author</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Web Address</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>&amp;Read</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>&amp;NotRead</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>From:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="fromDateEdit_3">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <item>
          <widget class="QLabel" name="toLabel_3">
           <property name="text">
            <string>To:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="toDateEdit_3">
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_20">
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="searchButton_3">
         <property name="text">
          <string>&amp;Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>searchLineEdit_3</tabstop>
  <tabstop>categoryComboBox_3</tabstop>
  <tabstop>radioButton</tabstop>
  <tabstop>radioButton_2</tabstop>
  <tabstop>fromDateEdit_3</tabstop>
  <tabstop>toDateEdit_3</tabstop>
  <tabstop>searchButton_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
